hs_core/views/resource_folder_hierarchy.py

```diff
@@ -172,7 +172,9 @@ def data_store_folder_unzip(request):
     try:
         unzip_file(user, res_id, zip_with_rel_path, bool_remove_original)
     except SessionException as ex:

```

Comment:
No, there could be other exceptions. It is nice to point out the main file override exception, though, since the error coming from iRODS is not very informative.